client/checkout/blocks/payment-processor.js

```diff
@@ -237,8 +235,9 @@ const PaymentProcessor = ( {
 		shouldSavePayment
 	);
 
-	const setPaymentInformationCompletionStatus = ( event ) => {
-		isPaymentInformationCompleteRef.current = event.complete;
+	const setHasLoadError = ( event ) => {
+		hasLoadErrorRef.current = true;
+		onLoadError( event );
```

Comment:
If the `onLoadError` is not provided, this call would throw an exception, because you can't call `event` on `undefined`.
What was the reason for removing the `noop` default assignment on L63?